src/OrchardCore.Modules/OrchardCore.OpenId/Services/OpenIdServerService.cs

```diff
@@ -78,7 +78,9 @@ private OpenIdServerSettings GetSettingsFromContainer(ISite container)
                 AuthorizationEndpointPath = "/connect/authorize",
                 LogoutEndpointPath = "/connect/logout",
                 TokenEndpointPath = "/connect/token",
-                UserinfoEndpointPath = "/connect/userinfo"
+                UserinfoEndpointPath = "/connect/userinfo",
+                IntrospectionEndpointPath = "/connect/introspect",
+                RevocationEndpointPath = "/connect/revoke",
```

Comment:
Please remove the unnecessary `,` and update the code comment to indicate the introspection and revocation endpoints are enabled by default too.